webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelect.stories.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import TypeAheadSelect from '.';
+import {
+  props,
+  propsWithNew,
+  propsWithPlaceholder,
+  propsWithMultiple,
+  propsWithClearButton,
+} from './TypeAheadSelect.fixtures';
+import Story from '../../../../../..//stories/components/Story';
+import storeDecorator from '../../../../../..//stories/storeDecorator';
+
+export default {
+  title: 'Components|TypeAheadSelect',
+  decorators: [storeDecorator],
+};
+
+export const defaultTypeAhead = () => (
+  <Story>
+    <TypeAheadSelect {...props} />
+  </Story>
+);
+
+defaultTypeAhead.story = {
+  name: 'Default',
+};
+
+export const withPlaceholder = () => (
+  <Story>
+    <TypeAheadSelect {...propsWithPlaceholder} />
+  </Story>
+);
+
+withPlaceholder.story = {
+  name: 'With Placeholder',
+};
+
+export const withNew = () => (
+  <Story>
+    <div>New options can be freely added that are not present in options</div>
+    <TypeAheadSelect {...propsWithNew} />
+  </Story>
+);
+
+withNew.story = {
+  name: 'New selections',
+};
+
+export const withMultiple = () => (
```

Comment:
For some reason this shows up only like 10px wide for me.  Probably a patternfly or Storybook issue though.

Reply:
Same - I have no idea :-/